style.css

```diff
@@ -0,0 +1,62 @@
+main {
+  background-color: #191765;
+  color: white;
+  font-family: 'Roboto', sans-serif;
+}
+
+a {
+  text-decoration: none;
+  color: white;
+  padding-left: 20px;
+  padding-top: 20px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav-menu {
+  padding-right: 20px;
+  padding-top: 20px;
+}
+
+.intro-photo {
+  background-image: url("images/Ellipse18.svg");
```

Comment:
The headline section should have a background image. Please use this image as a background for the headline section.
![back](https://user-images.githubusercontent.com/35504149/164798806-a92666e4-62ce-4704-b40e-2ee58166f4b7.png)